pkg/framework/custom_junit_reporter.go

```diff
@@ -83,7 +83,6 @@ func GenerateCustomJUnitReportWithConfig(report types.Report, dst string, config
 		test := JUnitTestCase{
```

Comment:
I have made modifications with Disabled in vendor/github.com/onsi/ginkgo/v2/reporters/junit_report.go 
but it's not reflected here.